src/components/Checkout/SocialShare.vue

```diff
@@ -649,5 +649,9 @@ $loan-triangle-size: rem-calc(12);
 	height: 1rem;
 	width: 1rem;
 	margin-right: 1rem;
+	color: $umbreon;
```

Comment:
How does $umbreon fit in with the rest of our color scheme? If it's something design is planning to use regularly we should add it to StyleguideColors.vue

Reply:
@eddieferrer I think we should avoid applying the umbreon color at this point as it looks like it will become basically a global override for everything in the content area of this page.

Reply:
Done, changed it back to kiva-text-dark